client/src/components/Onboarding.js

```diff
@@ -76,34 +76,54 @@ function Onboarding({children,title,activeStep,path,url}) {
 
     const [urlStore, setUrlStore] = useState("");
 
+    const alertContext = useContext(AlertContext)
+
+    const setAlert = (isOpen,message,type) => {
+        alertContext.setAlertStatus({
+            isOpen:isOpen,
+            message:message,
+            type:type
+          })
+    }
+
     const handleContinueClick = () => {
         let status;
         if (activeStep === 1){
-            if (url === "") return alert('empty url');
-            fetch(`/user/${url}/event-type`, {
-                method: "GET",
-                headers: {
-                    "Content-Type": "application/json"
-                }
-            })
-                .then(res => {
-                    status = res.status;
-                    if (status < 500) return res.json();
-                    else throw Error("Server error");
+            if (url === "") setAlert(true, "Url cannot be empty", "error");
+
+            else { 
+                fetch(`/user/${url}/event-type`, {
+                    method: "GET",
+                    headers: {
+                        "Content-Type": "application/json"
+                    }
                 })
-                .then(res => {
-                    if (status === 200) {
-                        if (res.unique === true) {
-                            setUrlStore(url);
-                            return history.push(path);
+                    .then(res => {
+                        status = res.status;
+                        if (status < 500) return res.json()
+                        else setAlert(true, "Server error", "error")
+                    })
+                    .then(res => {
+                        if (status === 200) {
+                            if (res.unique === true) {
+                                setUrlStore(url);
+                                return history.push(path);
+                            };
+                            return setAlert(true, "Url already exists", "error")
                         };
-                        return alert("Url exist")
-                    };
-                    throw Error("Failed to check");                
-                })
-                .catch(err => {
-                    alert(err.message);
-                });
+                        throw setAlert(true, "Unexpected error", "error")                
```

Comment:
Watch out here, you're throwing `setAlert`. I don't know if this is what you intended.